packages/service-library/tests/test_async_utils.py

```diff
@@ -151,6 +151,7 @@ async def target_function(the_param: Any) -> None:
     assert str(excinfo.value).startswith(message) is True
 
 
+@pytest.mark.flaky(max_runs=3)  # FIXME: ANE pls review and remove this flaky mark
```

Comment:
Have you got more insight about this?
This could be really really bad if it's the case. Why was it working before? What happened now?

Reply:
No. It simply fails randomly. You might want to review actions history for this test or run it many times until it fails?

Reply:
OK I've added the below PR to fix the issue. The timing being waited was too low.

- https://github.com/pcrespov/osparc-simcore/pull/68

Feel free to merge or change.